### PR TITLE
Added securitas direct documentation

### PR DESCRIPTION
**Pull request by nragon**
## Proposed change

Add documentation for the newly proposed [Securitas Direct](https://www.securitasdirect.pt) home security integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44448
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2107

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by nragon** [comment]
Follow up: https://github.com/home-assistant/home-assistant.io/pull/15986

File: source/_integrations/securitas_direct.markdown [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+```yaml
+# Example configuration.yaml entry
+securitas_direct:
+  username: USERNAME
+  password: PASSWORD
+  installation: INSTALLATION_NUMBER
+  code: 1234
+  country: PT
+  lang: pt
```
Comment (frenck) [inline source/_integrations/securitas_direct.markdown]: We no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/securitas_direct.markdown [hunk 2]
```diff
@@ -19,40 +19,19 @@ To integrate Verisure with Home Assistant, add the following section to your `co
...
+1. Click the Save button.
+
+| Attribute | Description | Required |
+| --------- | ----------- | ----------- |
+| `username` | Username for your Securitas Direct account.| `yes`
+| `password` | Password for your Securitas Direct account. | `yes`
+| `installation` | The number of your installation. To find information on installations run `pysecuritas -u USERNAME -p PASSWORD -c COUNTRY -l LANGUAGE INS`.| `yes`
+| `country` | Country identification (`PT`, `ES`, `FR`, `GB`, `IT`,...).| `default ES`
+| `lang` | Messages language (`pt`, `es`, `fr`, `en`, `it`,...).| `default es`
+| `code` | PIN code to activate or deactivate alarm.| `no`
```
Comment (frenck) [inline source/_integrations/securitas_direct.markdown]: If you want to provide such a list, please use a configuration_basic block (examples can be found in Lovelace cards documentation in this repository).

File: source/_integrations/securitas_direct.markdown [hunk 3]
```diff
@@ -19,40 +19,19 @@ To integrate Verisure with Home Assistant, add the following section to your `co
...
-code:
-  description: PIN code to activate or deactivate alarm.
-  required: false
-  type: integer
-{% endconfiguration %}
\ No newline at end of file
+## Configuration
+
+1. From Home Assistant, navigate to ‘Configuration’ then ‘Integrations’. Click the plus icon and type/select ‘Securitas Direct’.
+1. Choose your platform `Securitas Direct`.
```
Comment (frenck) [inline source/_integrations/securitas_direct.markdown]: Selecting securitas direct, was already in the previous step?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/securitas_direct.markdown (hunk 2 above)
Comment (nragon) [reply source/_integrations/securitas_direct.markdown]: I was following this example: https://www.home-assistant.io/integrations/fireservicerota/

File: source/_integrations/securitas_direct.markdown [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Securitas Direct can be integrated by adding the following `securitas_direct` section to your `configuration.yaml` file:
+
+## Configuration
+
+1. From Home Assistant, navigate to ‘Configuration’ then ‘Integrations’.
+1. Click the plus icon and type/select ‘Securitas Direct’.
+1. Enter your credentials and installation configuration.
+1. Click the Save button.
+
+{% configuration %}
```
Comment (frenck) [inline source/_integrations/securitas_direct.markdown]: This is a YAML configuration block.

If you want to provide such a list, please use a configuration_basic block (examples can be found in Lovelace cards documentation in this repository).

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/securitas_direct.markdown [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ha_domain: securitas_direct
+---
+
+The `securitas_direct` integration allows users to manage their Securitas Direct installation
+There is currently support for the following device types within Home Assistant:
+- **Alarm Control Panel**: Reports on the current alarm status and can be used to arm and disarm the system.
+
+## Configuration
+
+To integrate Verisure with Home Assistant, add the following section to your `configuration.yaml` file:
```
Comment (frenck) [inline source/_integrations/securitas_direct.markdown]: Something got lost here?

File: source/_integrations/securitas_direct.markdown [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+Securitas Direct can be integrated by adding the following `securitas_direct` section to your `configuration.yaml` file:
+
+## Configuration
```
Comment (frenck) [inline source/_integrations/securitas_direct.markdown]: Duplicate header

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/securitas_direct.markdown (hunk 5 above)
Comment (nragon) [reply source/_integrations/securitas_direct.markdown]: Yes :)

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Pull request closed without merging**